cypress/integration/advanced-functionalities/admin/system/messaging-queues/Create.spec.js

```diff
@@ -0,0 +1,31 @@
+/// <reference types="cypress" />
+const adminURL = Cypress.env('ADMIN_URL')
+const email = Cypress.env('USER_EMAIL')
+const password = Cypress.env('USER_PASSWORD')
+
+describe('Test for creating a messaging queue', () => {
+  before(() => {
+    if (!window.sessionStorage.getItem('auth.refresh-token')) {
+      cy.login({ email, password, url: adminURL })
+    }
+  })
+
+  context('Test for creating a messaging queue', () => {
+    it('should be able to create a messaging queue', () => {
+      cy.visit(adminURL + '/')
+      // We wait 3s in order the page to be fully loaded
+      cy.wait(3000)
+      cy.get('.nav-sidebar').contains('Messaging Queues').click()
+      // We wait 1s in order the page to be fully loaded
+      cy.wait(1000)
+      cy.get('[data-test-id="button-add"]').click()
+      cy.get('[data-test-id="input-name"]').type('TestQueue')
+      cy.get('[data-test-id="input-consumer"]').select('Store')
+      cy.get('[data-test-id="input-polling"]').type('5s')
+      cy.get('[data-test-id="button-submit"]').click()
+      // We confirm that the action was completed successfully
+      cy.get('.b-toast-success')
+      cy.get('[data-test-id="input-created-at"]').should('exist')
+    })
```

Comment:
Check if:
- the data is persisted
- deletedAt and updatedAt aren't shown
- delete, undelete and new btns aren't shown